### PR TITLE
Skycam link

### PR DESCRIPTION
**Pull request by dannygoldstein**
This small PR adds a field to the Telescope class that stores a link to the telescope's sky camera web viewer

### CODE REVIEW

**Comment by dannygoldstein** [comment]
@acrellin Test failures unrelated to PR

File: skyportal/models.py [hunk 1]
```diff
@@ -307,10 +307,12 @@ class SourceView(Base):
...
     nickname = sa.Column(sa.String, nullable=False)
-    lat = sa.Column(sa.Float, nullable=False)
-    lon = sa.Column(sa.Float, nullable=False)
-    elevation = sa.Column(sa.Float, nullable=False)
-    diameter = sa.Column(sa.Float, nullable=False)
+    lat = sa.Column(sa.Float, nullable=False, doc='Latitude in deg.')
+    lon = sa.Column(sa.Float, nullable=False, doc='Longitude in deg.')
+    elevation = sa.Column(sa.Float, nullable=False, doc='Elevation in meters.')
+    diameter = sa.Column(sa.Float, nullable=False, doc='Diameter in meters.')
+    skycam_link = sa.Column(sa.String, nullable=True,
```
[the remark this answers is not in the document]
Comment (dannygoldstein) [reply skyportal/models.py:314]: Ah nice! Didn't know about this

**Comment by stefanv** [comment]
Needs an API test, then good to go.

**Comment by dannygoldstein** [comment]
[ch1020]

**Comment by dannygoldstein** [comment]
Phew. Got my tests passing again. Thanks @acrellin

**Comment by dannygoldstein** [comment]
(@acrellin, needs your review and merge)

File: skyportal/tests/api/test_photometry.py [hunk 2]
```diff
@@ -845,6 +845,7 @@ def test_token_user_retrieve_null_photometry(upload_data_token, public_source,
                              'magsys': 'ab',
                              'filter': 'ztfg',
                              'group_ids': [public_group.id]
+
```
Comment (acrellin) [inline skyportal/tests/api/test_photometry.py:848]:
```suggestion
```

File: skyportal/tests/api/test_telescope.py [hunk 3]
```diff
@@ -12,7 +12,8 @@ def test_token_user_post_get_telescope(upload_data_token, public_group):
                              'lon': 0.0,
                              'elevation': 0.0,
                              'diameter': 10.0,
-                             'group_ids': [public_group.id]
+                             'group_ids': [public_group.id],
+                             'skycam_link': 'http://www.lulin.ncu.edu.tw/wea/cur_sky.jpg'
```
Comment (acrellin) [inline skyportal/tests/api/test_telescope.py:16]: Can we also test that it comes back as the expected value in a GET request?

**Comment by dannygoldstein** [comment]
@acrellin Ready for your review

**Pull request merged**